service/application/src/test/java/com/solace/maas/ep/event/management/agent/commandManager/SempPatchCommandManagerTest.java

```diff
@@ -0,0 +1,158 @@
+package com.solace.maas.ep.event.management.agent.commandManager;
+
+import com.solace.client.sempv2.ApiException;
+import com.solace.client.sempv2.api.RestDeliveryPointApi;
+import com.solace.client.sempv2.model.MsgVpnRestDeliveryPointRestConsumer;
+import com.solace.client.sempv2.model.MsgVpnRestDeliveryPointRestConsumerResponse;
+import com.solace.maas.ep.event.management.agent.TestConfig;
+import com.solace.maas.ep.event.management.agent.command.SempPatchCommandManager;
+import com.solace.maas.ep.event.management.agent.command.semp.SempApiProvider;
+import com.solace.maas.ep.event.management.agent.plugin.command.model.Command;
+import com.solace.maas.ep.event.management.agent.plugin.command.model.CommandType;
+import com.solace.maas.ep.event.management.agent.plugin.command.model.JobStatus;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.SpyBean;
+import org.springframework.test.context.ActiveProfiles;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.solace.maas.ep.common.model.SempEntityType.solaceRdpRestConsumer;
+import static com.solace.maas.ep.event.management.agent.plugin.command.model.SempPatchCommandConstants.SEMP_PATCH_DATA;
+import static com.solace.maas.ep.event.management.agent.plugin.command.model.SempPatchCommandConstants.SEMP_PATCH_ENTITY_TYPE;
+import static com.solace.maas.ep.event.management.agent.plugin.command.model.SempPatchCommandConstants.SEMP_PATCH_OPERATION;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.when;
+
+@ActiveProfiles("TEST")
+@EnableAutoConfiguration
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT, classes = TestConfig.class)
+public class SempPatchCommandManagerTest {
+
+    private final static String DIR_SEMP_RESOURCES = "src/test/resources/sempResponses/";
+    private final static String SEMP_RESPONSE_MISSING_RESOURCE = "sempResponseMissingResource.json";
+
+    @SpyBean
+    private SempPatchCommandManager sempPatchCommandManager;
+
+    private SempApiProvider sempApiProvider;
+
+
+    @BeforeEach
+    void reset() throws ApiException {
+        sempApiProvider = Mockito.mock(SempApiProvider.class);
+    }
+
+    @Test
+    void testPatchRdpRestConsumer() throws ApiException {
```

Comment:
Same request about using a `@Nested` class, in case we get to introduce more patch tests for non-RDP cases.